Neos.Flow/Configuration/Settings.Http.yaml

```diff
@@ -26,32 +26,34 @@ Neos:
 
       # Defines the PSR-15 middlewares that a request goes through in a flow
       middlewares:
+        # The following middleware components are executed first since they influence the incoming request or outgoing response
+        'standardsCompliance':
+          position: 'start 100'
+          middleware: 'Neos\Flow\Http\Middleware\StandardsComplianceMiddleware'
         'trustedProxies':
           position: 'start'
           middleware: 'Neos\Flow\Http\Middleware\TrustedProxiesMiddleware'
+        'session':
+          position: 'start'
```

Comment:
This leads to two middlewares sharing position "start" which is ambiguous. This makes it hard to reflect about the order if third party packages place middlewares relative to "session" as "trustedProxies" may or may not be executed before.